ts/modules/accessibility/focusBorder.ts

```diff
@@ -67,17 +71,70 @@ extend(H.SVGElement.prototype, {
             this.removeFocusBorder();
         }
         // Add the border rect
-        var bb = this.getBBox(),
+        const bb = this.getBBox(),
             pad = pick(margin, 3);
 
         bb.x += this.translateX ? this.translateX : 0;
         bb.y += this.translateY ? this.translateY : 0;
 
+        let borderPosX = bb.x - pad,
+            borderPosY = bb.y - pad,
+            borderWidth = bb.width + 2 * pad,
+            borderHeight = bb.height + 2 * pad;
+
+        // For text elements, apply x and y offset, #11397.
+        /**
+         * @private
+         * @function
+         *
+         * @param {Highcharts.SVGElement} text
+         *
+         * @return {TextAnchorCorrectionObject}
+         */
+        function getTextCorrection(text: Highcharts.SVGElement): TextAnchorCorrectionObject {
+            let posXCorrection = 0,
+                posYCorrection = 0;
+
+            if (text.attr('text-anchor') === 'middle') {
+                posXCorrection = H.isFirefox && text.rotation ? 0.25 : 0.5;
```

Comment:
The special treatment for Firefox, does it have a connection to this? https://github.com/highcharts/highcharts/blob/v8.0.4/ts/parts/SvgRenderer.ts/#L3705-L3729

Reply:
I don't think it is related to this case. In the RTL case, Firefox was changing the order of elements, and now it has a different baseline than other browsers.

Reply:
Makes sense, I think we have the same situation in the `textOutline` implementation.